source/scripts/Provider/EthProvider.ts

```diff
@@ -27,6 +27,19 @@ export const EthProvider = (host: string) => {
 
     if (!decodedTx) throw cleanErrorStack(ethErrors.rpc.internal());
 
+    console.log('params', params);
```

Comment:
Before we merge it remember removing the  console logs that are strict for testing